src/main/java/com/coreos/jetcd/data/Pair.java

```diff
@@ -0,0 +1,34 @@
+package com.coreos.jetcd.data;
```

Comment:
not sure if this belongs to here. this is not actually etcd kv pair but some internal helper structure, right? i feel it is confusing to put it under data.